packages/sdk/src/auth.ts

```diff
@@ -151,7 +151,13 @@ export class AuthManager {
       }
     }
     if (!token) {
-      await this.authentication.loginWithRedirect(opts);
+      const redirectOpts: RedirectLoginOptions = {
+        ...opts,
+        appState: {
+          returnTo: window.location.pathname + window.location.search
```

Comment:
template literal plz

Reply:
also returnTo is relative here, but above it included the origin

Reply:
removed origin above